.eslintrc.cjs

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
```

Comment:
https://github.com/eslint-community/eslint-plugin-n might also be a nice addition since we're in node space.

Reply:
Also, I would run prettier as a standalone script instead of as part of eslint: https://prettier.io/docs/en/integrating-with-linters#notes